### ISSUE TITLE
Add minimum required server version

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#29]
## Use case

At the moment, we have to be very careful with changes, as the app might be incompatible with a newer version of the server.

## Proposal

Add a new endpoint in the server, perhaps `required_version` that returns a version (we can use `get_version()` to generate it). The mobile app would load this on startup and compare its current version with the one downloaded. If the app's version is less than the required one, it would inform the user that he needs to update

**Comment by sandimilohanic** [comment]
@rolandgeider Is there already an endpoint or url to endpoint for checking to require version? If not yet, where should the endpoint be made, in the existing project or?

**Comment by rolandgeider** [comment]
no, there is currently no endpoint. If you want to also add it to the backend, go ahead, but I can also do it. The endpoint would return something looking like this "2.1a2.dev0", "2.1.3a2.dev0", "2.1.3b1", "2.4", "2.4.5". There are dart packages that can parse this

**Comment by sandimilohanic** [comment]
@rolandgeider I would be grateful if you could add an endpoint. thx

**Comment by rolandgeider** [comment]
Will do. You can work with hard coded values till then

**Comment by rolandgeider** [comment]
oh sorry, forgot to tell you, you need to use this URL https://wger-master.rge.uber.space/api/v2/min-app-version/

(wger.rge.uber.space has a different branch checked out)

**Comment by sandimilohanic** [comment]
Does it look good?  Do I need to add any more elements?

<img width="312" alt="Screenshot 2021-10-08 at 20 24 57" src="https://user-images.githubusercontent.com/7765448/136607989-15238e6b-fe7d-4b00-b3da-5c3267f3bf1f.png">
